### PR TITLE
Erik feature address entity

### PR DESCRIPTION
**Pull request by CodingErik**
Address Entity

### CODE REVIEW

**Comment by CodingErik** [comment]
Address Entity Feature

**Comment by wheelers-websites** [comment]
I want to start by saying that we are not expecting any additional commits. Feel free to make them if you wish, but that's not an expectation that we have. With that said I will go through your changes and leave technical comments on them.

File: src/main/java/com/bravo/user/dao/model/Address.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+@Table(name = "address")
+public class Address {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private String id;
+
+    @Column(name = "line1")
+    @NotBlank(message = "line1 is mandatory")
```
Comment (wheelers-websites) [inline src/main/java/com/bravo/user/dao/model/Address.java:22]: These types of validations belong on DTO classes, not DAO classes.

**Comment by CodingErik** [comment]
Thanks for the review Justin

File: src/main/java/com/bravo/user/model/dto/AddressReadDto.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+import java.time.LocalDateTime;
+
+@Data
+public class AddressReadDto {
+
+    private String id;
+    private String address;
+    private String line1;
+    private String line2;
```
Comment (tlynema-bravolt) [inline src/main/java/com/bravo/user/model/dto/AddressReadDto.java:13]: If a reviewer sees line1 and line2, they may not know what this is.  Even inside of an address-related object.  I've seen all the way up to 5 of these in codebases that handle international addresses :) .

File: src/main/resources/data.sql [hunk 3]
```diff
@@ -9,6 +10,18 @@ create table user (
     updated timestamp not null default current_timestamp()
 );
 
+create table address (
+    id int primary key AUTO_INCREMENT,
+    line1 varchar(100) not null,
+    line2 varchar(100),
+    city varchar(100) not null,
+    state varchar(100) not null,
+    zip varchar(100) not null,
```
Comment (tlynema-bravolt) [inline src/main/resources/data.sql:19]: zip codes don't go to a length 100.

Comment (tlynema-bravolt) [reply src/main/resources/data.sql:19]: And you had this as an int in the dao so not 100 there either.  It's nitpicky though.

File: src/main/resources/data.sql [hunk 4]
```diff
@@ -610,4 +623,507 @@ insert into user (id, first_name, middle_name, last_name, phone_number) values
...
+        ('2959 Dierdre Shore','Apt. 885','New Davinachester','IL','90511','71e0b8e3-24d3-491f-af6a-672b2930e98f'),
+        ('540 Pamala Locks','Suite 255','Alishaburgh','MI','42777','7264af5a-6ffa-4c6b-ba05-479b69d7f1f4'),
+        ('203 Kuhlman Stream','Apt. 586','Trompchester','AL','34066','72e78bd6-43bd-43c8-b2db-277118f6120c'),
+        ('5284 Dianna Villages','Suite 950','Lake Carey','NJ','93395','735a5f74-8d84-438e-8b95-883230ccbb97'),
+        ('19323 Cassey Vista','Apt. 132','North Reginia','ME','31024','73883332-94b0-401f-bdb6-dbc7e66eff2e'),
+        ('342 Augustina Squares','Apt. 465','Lake Blondellchester','NM','34125','7390550f-2307-4c73-b1e3-99d11f58a5b5'),
+        ('544 Doyle Wells','Apt. 957','West Halley','TN','40077','73a3808a-bcae-4701-b0cc-fa3ca26e7c16');
+
+
+
```
Comment (tlynema-bravolt) [inline src/main/resources/data.sql:991]: Like the samples to some extent.  In testing, they should all have a purpose.  e.g. ensure that all states work.

**Pull request closed without merging**